### PR TITLE
Added setRetroMake to fix #1

### LINKED ISSUE #1: Allow specifying make function
For the retro-contest I need to be able to use a different environment creator

Normal
```
import retro
env = retro.make(game, **kwargs)
```

Contest
```
from retro_contest.local import make
env = make(game, **kwargs)
```

### PR DESCRIPTION
**Pull request by frenchie4111**

### CODE REVIEW

**Comment by frenchie4111** [comment]
Usage:
```
import retrowrapper
from retro_contest.local import make
retrowrapper.setRetroMake( make )
env1 = retrowrapper.RetroWrapper(
    game='SonicTheHedgehog2-Genesis', 
    state='MetropolisZone.Act1'
)
env2 = retrowrapper.RetroWrapper(
    game='SonicTheHedgehog2-Genesis', 
    state='MetropolisZone.Act2'
)
```

File: retrowrapper.py [hunk 1]
```diff
@@ -3,14 +3,21 @@
 """
 import multiprocessing
 import retro
+import gc
+import time
+
+MAKE_RETRIES = 5
+
+def setRetroMake( new_retro_make_func ):
```
Comment (MaxStrange) [inline retrowrapper.py]: Please rename to 'set_retro_make'

File: retrowrapper.py [hunk 2]
```diff
@@ -3,14 +3,21 @@
 """
 import multiprocessing
 import retro
+import gc
+import time
```
Comment (MaxStrange) [inline retrowrapper.py]: Remove unused import

**Review by MaxStrange — CHANGES_REQUESTED** [review]
Cool. Could you also update the README with an additional example with the new usage?

**Comment by frenchie4111** [comment]
Changes complete

**Review by MaxStrange — APPROVED** [review]

**Pull request merged**